schema/project-level/project-schema.json

```
@@ -2336,6 +2351,92 @@
         }
       },
       "minProperties": 1
+    },
+    "Meeting": {
+      "title": "Meeting",
+      "description": "The details of a meeting.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "ID",
+          "description": "A locally unique identifier for the meeting.",
+          "type": [
+            "string"
+          ]
+        },
+        "date": {
+          "title": "Date",
+          "description": "The date of the meeting.",
+          "type": [
+            "string"
+          ],
+          "format": "date-time"
+        },
+        "address": {
+          "title": "Address",
+          "description": "The address of the meeting.",
+          "$ref": "#/definitions/Address"
+        },
+        "numberOfParticipants": {
+          "title": "Participant count",
+          "description": "The number of people in attendance at the meeting.",
+          "type": "integer"
+        },
+        "publicOfficial": {
```

And corresponding changes elsewhere. The "official" is always the person, so we need to refer to this as the "office" (i.e. the position), otherwise it's confusing.

```suggestion
        "publicOffice": {
```